src/rules/noStringThrowRule.ts

```diff
@@ -73,10 +73,20 @@ function walk(ctx: Lint.WalkContext<void>): void {
         if (isThrowStatement(node)) {
             const { expression } = node;
             if (expression !== undefined && isString(expression)) {
-                ctx.addFailureAtNode(node, Rule.FAILURE_STRING, [
+                const fix = [
                     Lint.Replacement.appendText(expression.getStart(sourceFile), "new Error("),
                     Lint.Replacement.appendText(expression.getEnd(), ")"),
-                ]);
+                ];
+
+                const hasWhitespaceAfterThrow = /^throw\s+/.test(node.getText());
```

Comment:
We normally try not to use regular expressions on `getText` to check for positioning. It's a bit cleaner and faster to use `.pos` when possible.

How about checking `node.expression.pos`? It should be `"throw.length"` greater than `node.pos`.

Reply:
I ended up using the `getStart()` method and doing a quick calculation to check if there are any characters preceding the string literal directly after the throw.